### PR TITLE
Fix hires geocoder health check

### PR DESCRIPTION
**Pull request by rafatower**
This stopped working because of the refactor plus the way it avoided using the `Cartodb` module and other parts of rails framework. In the case of configs they were always passed through constructors, cluttering all the code.

In order to fix it:
- Added a `GeocoderConfig` singleton
- Fixed the geocoder_test script by injecting config overrides through the GeocoderConfig

Anyone from @CartoDB/backend please review

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5026/))

File: services/geocoder/lib/hires_batch_geocoder.rb [hunk 1]
```diff
@@ -163,8 +164,7 @@ def result
     private
 
     def config
```
Comment (Kartones) [inline services/geocoder/lib/hires_batch_geocoder.rb]: dunno if is worth keeping the `@config`, as the GeocoderConfig is already a singleton. If is for testing purposes you can still stub the `config` method, but I'd probably just do:
```
def config
  GeocoderConfig.instance.get
end
```

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5027/))

**Comment by Kartones** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5028/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5030/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5034/))

**Pull request merged**